### DISCUSSION TITLE
本地运行exe, 还是无法解锁灰色

### DISCUSSION THREAD

**Discussion by msccreater** [1715173329/UnblockNeteaseMusic#1083] [Q&A]
### Bug 描述

本地运行exe, 还是无法解锁灰色

### 日志内容

直接下载的exe双击运行的, 没有LOG

### 网易云音乐版本号

2.10.8

### 操作系统

win11

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by ReviveUnblockNCMInstaller** [comment]
> 直接下载的exe双击运行的

你是只打开了 exe，没有做任何配置？

**Comment by shiyaow2** [comment]
![image](https://user-images.githubusercontent.com/112105993/233802050-78ded456-f2a1-4d5e-bd2e-feb99d14a4b4.png)
你在网易云里调代理设置了吗？
